cfme/services/myservice/ui.py

```diff
@@ -30,29 +45,9 @@ def is_displayed(self):
 
```

Comment:
Need to fix old calls to MyServicesView.configuration, since it moved to toolbar (in is_displayed above, GH won't let me comment directly on that line.

```python
>   request.addfinalizer(lambda: _cleanup(appliance, provision_request))

cfme/tests/services/test_provision_stack.py:205: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
cfme/tests/services/test_provision_stack.py:150: in _cleanup
    myservice.delete()
/usr/lib/python2.7/site-packages/sentaku/context.py:132: in __call__
    return bound_method(*k, **kw)
cfme/services/myservice/ui.py:255: in delete
    assert view.is_displayed
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <cfme.services.myservice.ui.MyServicesView object at 0x7fbc22f1dc10>

    @property
    def is_displayed(self):
        return (
            self.in_myservices and
>           self.configuration.is_displayed and not
            self.myservice.is_dimmed)
E       AttributeError: 'MyServicesView' object has no attribute 'configuration'

cfme/services/myservice/ui.py:43: AttributeError
```

Reply:
Doh. Knew there'd be at least one reference I didn't catch.

Reply:
Two more references missed here , Raising PR https://github.com/ManageIQ/integration_tests/pull/6918